OpenNefia.Content/UI/Hud/Widgets/HudClockWidget.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Love;
+using OpenNefia.Core.Rendering;
+using OpenNefia.Core.UI.Element;
+using OpenNefia.Content.Prototypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static OpenNefia.Core.Rendering.AssetInstance;
+using OpenNefia.Content.World;
+using OpenNefia.Core.IoC;
+
+namespace OpenNefia.Content.Hud
+{
+    public class ClockHand : BaseDrawable
+    {
+        // Angle in radians
+        private float Angle;
+        private SpriteBatch Batch;
+        public ClockHand()
+        {
+            var asset = Assets.Get(Protos.Asset.ClockHand);
+            var parts = new List<AssetBatchPart>{ new("0", 0, 0) };
```

Comment:
Could you do `new AssetDrawable(Protos.Asset.ClockHand, regionId: "0")` instead?

Reply:
`Graphics.Draw()` doesn't take AssetDrawables, also AssetDrawables don't seem to support origin offsets, making the rotation very difficult.

Reply:
changed in 54e3065